WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -256,7 +256,7 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 			catch (Exception ex)
 			{
 				roundState.Log(LogLevel.Warning, $"Waiting for the round to end failed with: '{ex}'.");
-				throw new UnknownRoundEndingException(signedCoins, outputTxOuts.Select(o => o.ScriptPubKey).ToImmutableList(), ex);
+				throw new UnknownRoundEndingException(signedCoins, outputTxOuts.Select(o => o.ScriptPubKey).ToImmutableList(), unsignedCoinJoin, ex);
```

Comment:
At this point we don't know whether everybody signed or not. The only thing we know is that we don't know so, whatever we assume is wrong. 

In case the transaction is broadcasted, if we assume it was broadcasted then we will receive the real transaction via p2p or in a block but,
In case the transaction is NOT broadcasted, if we assume it was broadcasted then we will have a a wrong utxoset (wrong balance).

How is it supposed Wasabi to correct a wrongly assumed coinjoin? Is there any mechanism to revert it?

Reply:
Reverted that change.